src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -15,7 +15,16 @@
     <Compile Include="System\Net\Quic\Implementations\Mock\*.cs" />
     <Compile Include="System\Net\Quic\Implementations\MsQuic\*.cs" />
     <Compile Include="System\Net\Quic\Implementations\MsQuic\Internal\*.cs" />
-    <Compile Include="System\Net\Quic\Implementations\MsQuic\Interop\*.cs" />
+    <Compile Include="System\Net\Quic\Implementations\MsQuic\Interop\MsQuicAlpnHelper.cs" />
```

Comment:
Is this change of csproj on purpose?

Reply:
yes. The wildcard would always include all flavors. With the change we include common files here explicitly and then one specific for each platform.